tests/src/Kernel/BackendIngestionTest.php

```diff
@@ -278,15 +278,13 @@ public function testDeleteItems(): void {
   public function testDeleteAllIndexItems(): void {
     $this->assertServiceMockCalls(EuropaSearchMockServerConfigOverrider::ENDPOINT_INGESTION_DELETE, 0, 0);
     $this->backend->deleteAllIndexItems($this->index);
-    $this->assertServiceMockCalls(EuropaSearchMockServerConfigOverrider::ENDPOINT_INGESTION_DELETE, 5, 5);
+    $this->assertServiceMockCalls(EuropaSearchMockServerConfigOverrider::ENDPOINT_INGESTION_DELETE, 15, 15);
     // Compare sent data with received data.
     $requests = $this->getServiceMockRequests(EuropaSearchMockServerConfigOverrider::ENDPOINT_INGESTION_DELETE);
-    $this->assertCount(5, $requests);
-    $this->assertDeletedItem($requests[0], 1);
-    $this->assertDeletedItem($requests[1], 2);
-    $this->assertDeletedItem($requests[2], 3);
-    $this->assertDeletedItem($requests[3], 4);
-    $this->assertDeletedItem($requests[4], 5);
+    $this->assertCount(15, $requests);
+    for ($i = 1; $i <= 5; $i++) {
```

Comment:
Probably it should be 15 items. But looks like there are discrepancies in the number of items.
```suggestion
    for ($i = 1; $i <= 15; $i++) {
```

Reply:
The mock returns the same result for the search test which created 15 items. In this test though we only create 5 items, so we only test those, otherwise it would fail with an invalid scenario.